### PR TITLE
Add Swedish translation

### PR DESCRIPTION
**Pull request by franpoli**
#### What does this implement/fix?

Add a Swedish translation

### CODE REVIEW

File: _translations/index_se.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+ett rättvist, opartiskt sätt och bör inte ge efter för yttre
+sociala påtryckningar. Vi uppmanar FSF att objektivt överväga argumenten
+mot RMS och att verkligen förstå innebörden av hans
+ord och handlingar.
+
+Historiskt har RMS uttryckt sina åsikter på sätt som upprör
+många människor. Han är vanligtvis mer fokuserad på den
+filosofiska grunden, att sträva efter den objektiva sanningen
+och språklig purism, samtidigt som han understryker
+människors känslor i frågor som han uttalar sig om.
```
Comment (jobbautista9) [inline _translations/index_se.md:34]: According to machine translation: _while emphasizing
people's feelings on issues he speaks about._

This seems the opposite of _while underemphasising people’s feelings on matters he’s commenting on._

File: README_SE.md [hunk 2]
```diff
@@ -1,49 +1,43 @@
...
-name: Exempel på namn (bra företag)
-länk: https://github.com/example_username
+name: Anders Svensson (God Företag AB)
+link: https://github.com/anders_svensson
 ```
 
-Använd inte `<>` i den här filen, och inte heller icke-ascii-symboler i filnamnet.
-Om du använder din e-postadress som länk ska du sätta `mailto:` före.
-Om du har möjlighet, använd ditt riktiga namn och lägg till projekt och anslutna organisationer inom parentes.
+Använd inte `<>` eller icke-ASCI-symboler i filnamnet.
```
Comment (jobbautista9) [inline README_SE.md:19]: Shouldn't this be _ASCII_, not _ASCI_?

File: README_SE.md [hunk 3]
```diff
@@ -1,49 +1,43 @@
...
+- **IRC:** #free-rms on [freenode](https://freenode.net)
 - **Discord:** https://discord.gg/7FWkxG4CsU
-- **Telegram:** https://t.me/free_rms
-
-## Licensiering
-Koden i arkivet är licensierad under GPL-3.0-only.
-
-Bilderna i mappen `assets` är licensierade under [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/legalcode). De sociala förhandsgranskningsbilderna är baserade på https://commons.wikimedia.org/wiki/File:Richard_Matthew_Stallman.jpeg, publicerad under CC BY-SA 3.0, som ursprungligen publicerades som omslaget till O'Reilly-boken `Free as in Freedom: Richard Stallman's Crusade for Free Software by Sam Williams`, publicerad den 1 mars 2002 under GFDL.
-
-Signaturer är inte upphovsrättsligt skyddade.
```
Comment (jobbautista9) [inline README_SE.md:49]: Why remove the translation of the licensing notice?

**Review by jobbautista9 — COMMENTED** [review]
Hi, thanks for taking the time to translate the letter! Here's some comments of mine.

I'm not a native speaker of Swedish, so I have to rely on machine translation, so I'd like someone who is Swedish to review this translation.

File: README_SE.md (hunk 3 above)
Comment (franpoli) [reply README_SE.md:49]: The licensing was not mandatory as it was explained somewhere in a discussion thread, so I did the same as I could observe on other translations.

Comment (jobbautista9) [reply README_SE.md:49]: It's not mandatory, but it doesn't have to be removed either when it's already there though.

If removing this makes translation easier, then I will accept it.

**Comment by franpoli** [comment]
I fixed the licensing. Let me know if you have further concerns.

**Comment by jobbautista9** [comment]
Thanks. Please see the other issues above that I pointed out (i.e. ASCII and emphasized)

**Comment by jobbautista9** [comment]
@dudugus Do you approve this translation?

**Comment by franpoli** [comment]
Thank you for your vigilance. I got the proper sense for "underemphasising" in the sentence.

**Comment by jobbautista9** [comment]
Thanks, I don't think we need some native speaker help anymore, now that the underemphasizing issue has been solved according to machine translation.

We can always fix it in another PR if there are mistakes. :)

**Pull request merged**